api/opencl_runtime_layer.asciidoc

```diff
@@ -16098,20 +16158,17 @@ include::{generated}/api/version-notes/CL_MUTABLE_COMMAND_COMMAND_TYPE_KHR.ascii
 | {CL_MUTABLE_DISPATCH_PROPERTIES_ARRAY_KHR_anchor}
 
 include::{generated}/api/version-notes/CL_MUTABLE_DISPATCH_PROPERTIES_ARRAY_KHR.asciidoc[]
-  | {cl_ndrange_kernel_command_properties_khr_TYPE}[]
-    | Return the properties argument specified on _command_ recording with
-      {clCommandNDRangeKernelKHR}.
+  | {cl_command_properties_khr_TYPE}[]
```

Comment:
Should we consider renaming this query to something a bit more general now that other command buffer commands can also have optional properties arguments?  For example, should we consider `CL_MUTABLE_COMMAND_PROPERTIES_ARRAY_KHR` rather than the mutable dispatch-specific `CL_MUTABLE_DISPATCH_PROPERTIES_ARRAY_KHR`?

Reply:
That's a good idea, I've made that change as a new commit so it can be reviewed in isolation.